.github/workflows/local_tests.yaml

```
@@ -0,0 +1,42 @@
+name: Local chain tests
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+jobs:
+    tests:
+      strategy:
+          fail-fast: true
+      runs-on: ubuntu-latest
+      name: Build and Test
+      steps:
+      - uses: actions/checkout@v3
+        name: Check out repository
+      - name: Checkout submodules
+        run: git submodule update --init --recursive
+      - uses: actions/setup-node@v2
+        name: Set up Node.js
+        with:
+          node-version: "lts/gallium"
+      - name: Install graph CLI
+        run: |
+          npm install -g @graphprotocol/graph-cli
```

Using a pipe is useful when you want to write over multiple lines. No need here (and many other places).
More info about this here: https://yaml-multiline.info/
```suggestion
        run: npm install -g @graphprotocol/graph-cli
```